projects/js-toolkit/packages/js-toolkit-core/src/project/liferayCli/Start.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import LiferayJson from '../../schema/LiferayJson';
+import Project from './Project';
+
+export default class Start {
+	readonly port: number;
+
+	constructor(_project: Project, liferayJson: LiferayJson) {
+		this.port = liferayJson.start?.port || 8081;
+	}
+}
```

Comment:
I'm curious why you went with a class for everything? Seems like this could have just been a plain object

Reply:
Mainly because I copied it from bundler2 and bundler3, and it's how it was structured there. In those two tools, the configuration options grew big enough to make it worth separating it into functional scopes (just to make the code easier to navigate).

So, even if it can be one single class now, I'm almost sure it will grow. If it doesn't grow, this will be a YAGNI pattern but still, I don't think it hurts separating things by scopes (IMO), so... :shrug:

Reply:
Also, you can write things like `project.deploy.dir` and `project.build.dir` which look beatiful :heart_eyes: :joy: